rebac-admin-backend/Makefile

```diff
@@ -22,5 +22,9 @@ mocks:
 
 # Run tests
 test: mocks
-	go test ./...
+	go test ./... -cover -coverprofile coverage_source.out
+	# this will be cached, just needed to the test.json
+	go test ./... -cover -coverprofile coverage_source.out -json > test_source.json
+	cat coverage_source.out | grep -v "mock_*" | tee coverage.out
+	cat test_source.json | grep -v "mock_*" | tee test.json
```

Comment:
Since we can also run this target on our local machines, we should add `test_source.json`, `coverage_source.out`, `coverage.out` and `test.json` to the `.gitignore` file.

Reply:
The *.out files are already in the .gitignore
I'll add the other ones, thanks